src/platform/site-wide/tests/MegaMenu.unit.spec.jsx

```diff
@@ -0,0 +1,259 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { axeCheck } from 'platform/forms-system/test/config/helpers';
```

Comment:
Changing the location of the `axeCheck` import was the only "code change" I made to the migrated megamenu files.